terraform/grafana-alerts/alert-rules-trading-modes.tf

```diff
@@ -9,6 +9,7 @@ resource "grafana_rule_group" "trading_modes" {
     content {
       name      = "Trading Mode Alert [${title(rule.value)}]"
       condition = "isTradingHalted"
+      for       = "5m"
```

Comment:
cooldown for mediandeltabreaker is always 15-30 minutes, so I think it would make sense for this to be at least 15 as well, since before that the value of the metric won't change. It's slightly different for the valuedeltabreaker but I think using the same pending period for both it's fine.